web/src/components/layout/authenticated-layout.tsx

```diff
@@ -43,19 +44,22 @@ export function AuthenticatedLayout({ children }: AuthenticatedLayoutProps) {
   ];
 
   const workspaceNavigation: SidebarNavigationItem[] = [];
-
   const projectsNavigation: SidebarNavigationItem[] = [];
 
   return (
     <AddTodoFormProvider>
       <EditTodoFormProvider>
         <TodoSheetProvider>
           <SidebarProvider>
+            
+            <TopProgressBar />
```

Comment:
The `TopProgressBar` should be in the `__root.tsx` file, so it is applied on all pages, not just on authenticated ones.